src/index.html

```diff
@@ -1,14 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no,initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="column">
+        <div class="group__box1">
+          <div class="box box1">1</div>
+          <div class="box box2">2</div>
+          <div class="box box3">3</div>
+          <div class="box box4">4</div>
+          <div class="box box5">5</div>
+          <div class="box box6">6</div>
+        </div>
```

Comment:
It should be only one group of boxes and it should change according to the screen size.
Use media-queries for this.
For screen with a width up to 300px the blocks should be set up from top to bottom (as in your `group__box3`)
for width from 301px to 899px it should be two columns and three rows (as in your `group__box2`)
and for a width 900px and more it should be two rows with three columns (as in your `group_box1`).
All these blocks should be adaptive and change in size according to the screen width